miden-crypto/src/dsa/rpo_falcon512/tests/data.rs

```diff
@@ -1749,3 +1749,80 @@ pub(crate) static SK_POLYS: [[[i16; 512]; 4]; NUM_TEST_VECTORS] = [
         ],
     ],
 ];
+
+pub(crate) const DETERMINISTIC_SIGNATURE: [u8; 1524] = [
```

Comment:
Could we add a comment explaining where this data is coming from (or how it was computed)?

Reply:
Don't have a strong opinion here. But if we do end up keeping it, I'd add a comment saying that this was pre-computed on such-and-such hardware to tests consistency across different architectures (or something to this effect).